lib/forge_sdk/conn/supervisor.ex

```diff
@@ -34,6 +34,10 @@ defmodule ForgeSdk.ConnSupervisor do
     DynamicSupervisor.which_children(__MODULE__)
   end
 
+  def get_names do
+    Enum.map(children(), fn {_, p, _, _} -> Process.info(p, :registered_name) end)
```

Comment:
iex(7)> Process.info(pid(0,0,0))[:registered_name]
:init

iex(6)> Process.info(pid(0,0,0),:registered_name)
{:registered_name, :init}

You probably want to do a pattern matching to get the name if this function is expecting to return a list of names.